### PR TITLE
Fix address lookup service redirecting back to new journey

### PR DESCRIPTION
**Pull request by danielmaltz1**

### CODE REVIEW

**Review by SiddharthAtos — APPROVED** [review]

File: app/connectors/AddressLookupConnector.scala [hunk 1]
```diff
@@ -43,7 +43,12 @@ class AddressLookupConnector @Inject()(val appConfig: FrontendAppConfig,
     s"${baseUrl}/api/v2/confirmed?id=$id"
   }
 
-  def continueUrl(isAgent: Boolean, isChange: Boolean): String = if (isAgent) controllers.incomeSources.add.routes.AddBusinessAddressController.agentSubmit(None, isChange = isChange).url else controllers.incomeSources.add.routes.AddBusinessAddressController.submit(None, isChange = isChange).url
+  def continueUrl(isAgent: Boolean, isChange: Boolean): String = (isAgent, isEnabled(IncomeSourcesNewJourney)) match {
```
Comment (avelx) [inline app/connectors/AddressLookupConnector.scala:46]: I guess this changes would imply changes in the tests?

Comment (danielmaltz1) [reply app/connectors/AddressLookupConnector.scala:46]: We stub the API response with the correct url in the tests already

Comment (avelx) [reply app/connectors/AddressLookupConnector.scala:46]: I see, we are not testing this method in any test; 
we might need to be consistent as in some connectors we are testing methods returning urls, but in some not

Comment (danielmaltz1) [reply app/connectors/AddressLookupConnector.scala:46]: ATs should cover this, as they check whether the user has been redirected to the correct location

**Review by avelx — APPROVED** [review]

**Pull request merged**